examples/feature_convert/step/partcad.yaml

```
@@ -0,0 +1,22 @@
+desc: PartCAD example project to demonstrate parts defined using STL files.
+
+docs:
+  usage: |
+    ```shell
+    pc inspect cube
+    ```
+
+parts:
+  bolt:
+    type: step
+    desc: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)